### ISSUE TITLE
cpu issue

### ISSUE THREAD

**Issue by ShinAbo** [rowak/nanoleaf-desktop#15]
Just started using the app and its good, but i noticed it uses 60% of my cpu ? is that normal ? CPU temp goes to 60C° when using it (my CPU is i7 8700)

**Comment by rowak** [comment]
This is not normal when the app is idle. However, you may experience this cpu usage when using ambient lighting mode or Spotify visualizer mode depending on your settings. Are you using either of these modes when the cpu usage is at 60%? If so, what are your "update delay" settings for these modes? You can likely fix the issue by moving the slider closer to the right side.

**Comment by ShinAbo** [comment]
Yes the CPU spike only happens when i use ambient lightning . slider was in the middle, moving it to the right decreased CPU usage by 2% (went from 60% to 58%)

**Comment by rowak** [comment]
That's weird. Are you controlling multiple Nanoleaf devices grouped together, or only one? What firmware are your device(s) on? Also, try using an older release ([v0.7.1](https://github.com/rowak/nanoleaf-desktop/releases/tag/v0.7.1)) to see if you still have the cpu problems.

**Comment by melgu** [comment]
Can confirm. It almost fully pegs 8 of my 16 threads no matter the slider position.
